sdk/src/feature_set.rs

```diff
@@ -991,6 +995,7 @@ lazy_static! {
         (enable_tower_sync_ix::id(), "Enable tower sync vote instruction"),
         (chained_merkle_conflict_duplicate_proofs::id(), "generate duplicate proofs for chained merkle root conflicts"),
         (reward_full_priority_fee::id(), "Reward full priority fee to validators #34731"),
+        (abort_on_invalid_curve::id(), "Abort when elliptic curve syscalls invoked on invalid curve id")
```

Comment:
PR or SIMD number missing

Reply:
Added!

Reply:
don't see it, I meant the description string here.

Reply:
Oh no, sorry I think I overwrote it when I rebased 🙏. I added it now.